app/code/RLTSquare/Unit3/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <!-- START: Tab Unit3 -->
+        <tab id="rltsquare" translate="label" sortOrder="10">
+            <label>RLTSquare</label>
+        </tab>
+        <!-- START: section Unit3 -->
+        <section id="unit3" showInDefault="1" showInWebsite="0" showInStore="1" sortOrder="10" translate="label">
+            <label>Unit3</label>
+            <tab>rltsquare</tab>
+            <resource>RLTSquare_Unit3::rltsquare_config</resource>
+            <!-- START: group Settings -->
+            <group id="Settings" translate="label" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="1">
+                <label>Settings</label>
+                <!-- START: field configuration -->
+                <field id="configuration" showInDefault="1" showInWebsite="0" showInStore="1" sortOrder="10"
+                       translate="label" type="select">
+                    <label>Enable Unit3 Configuration</label>
+                    <comment>select yes or no to enable/disable configuration</comment>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                </field>
+                <!-- END: field configuration -->
+                <!-- START: field username -->
+                <field id="username" showInDefault="1" showInWebsite="0" showInStore="1" sortOrder="10" translate="label" type="text">
+                    <label>Username</label>
+                    <comment>Enter a username</comment>
+                </field>
+                <!-- END: field username -->
+                <!-- START: field password -->
+                <field id="password" showInDefault="1" showInWebsite="0" showInStore="1" sortOrder="10" translate="label" type="obscure">
```

Comment:
There should be separate field for staging password, separate for production, etc... as we want to preserve any info entered against certain environment.

```
<field id="environment" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
    <label>Enabled</label>
    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
</field>
<field id="staging_pass" translate="label" type="text" sortOrder="15" showInDefault="1"   showInWebsite="1" showInStore="1">
    <label>Password</label>
    <depends>
        <field id="environment">staging</field>
    </depends>
    <validate>required-entry validate-password no-whitespace</validate>
    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
</field>
<field id="production_pass" translate="label" type="text" sortOrder="15" showInDefault="1"   showInWebsite="1" showInStore="1">
    <label>Password</label>
    <depends>
        <field id="environment">production</field>
    </depends>
    <validate>required-entry validate-password no-whitespace</validate>
    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
</field>
```

Within the helper getPassword, you would still return single pass based of selected environment.